src/dev/jest/config.js

```diff
@@ -78,7 +78,7 @@ export default {
   ],
   coverageDirectory: '<rootDir>/target/kibana-coverage/jest',
   coverageReporters: !!process.env.CODE_COVERAGE ? ['json'] : ['html', 'text'],
-  moduleFileExtensions: ['js', 'json', 'ts', 'tsx'],
+  moduleFileExtensions: ['js', 'json', 'ts', 'tsx', 'node'],
```

Comment:
Without this change, the build was failing with the following error:

> Cannot find module './build/Release/re2' from 're2.js'
> 
>     However, Jest was able to find:
>         'build/Release/re2.node'
> 
>     You might want to include a file extension in your import, or update your 'moduleFileExtensions', which is currently ['js', 'json', 'ts', 'tsx'].
> 
>     See https://jestjs.io/docs/en/configuration#modulefileextensions-array-string

'node' is already in the [defaults](https://jestjs.io/docs/en/configuration#modulefileextensions-arraystring), and I'm not sure why we're overriding the defaults. @elastic/kibana-operations mind double-checking this?

Reply:
I expect the defaults have changed over the years to include `ts`/`tsx`, but didn't originally, which is probably why we're overriding them. LGTM, though I'm not opposed to just deleting the override and using the defaults either (in this and other files too).